src/electionguard-ui/ElectionGuard.UI/ViewModels/TallyProcessViewModel.cs

```diff
@@ -1,24 +1,39 @@
-﻿using CommunityToolkit.Mvvm.Input;
+﻿using System.Threading.Tasks;
+using CommunityToolkit.Mvvm.Input;
 using ElectionGuard.UI.Lib.Extensions;
 using ElectionGuard.UI.Models;
 using ElectionGuard.UI.Services;
 
 namespace ElectionGuard.UI.ViewModels;
 
 [QueryProperty(CurrentTallyIdParam, nameof(TallyId))]
+[QueryProperty(MultiTallyIdParam, nameof(MultiTallyId))]
 public partial class TallyProcessViewModel : BaseViewModel
 {
-    public const string CurrentTallyIdParam = "TallyId";
+    public const string CurrentTallyIdParam = nameof(TallyId);
+    public const string MultiTallyIdParam = nameof(MultiTallyId);
 
     [ObservableProperty]
     private string _tallyId = string.Empty;
 
+    [ObservableProperty]
+    private List<string> _multiTallyIds = new();
+
+    [ObservableProperty]
+    private string _multiTallyId = string.Empty;
+
+    [ObservableProperty]
+    private MultiTallyRecord? _currentMultiTally;
+
     [ObservableProperty]
     private bool _canUserJoinTally;
 
     [ObservableProperty]
     private bool _canUserStartTally;
 
+    [ObservableProperty]
+    public bool _isMultiTally = !string.IsNullOrEmpty(MultiTallyId);

```

Comment:
This can be used by the UI for the determining to show the list of tallies, so it was made as an ObservableProperty.  Could be changed to an expression bodied property and then call the onPropertyChanged manually.